src/metrics.go

```diff
@@ -161,7 +162,11 @@ func getRawMetrics(info string) (map[string]interface{}, map[string]map[string]i
 			}
 		} else {
 			value := strings.TrimSuffix(parts[1], "\r\n")
-			metrics[parts[0]] = asValue(value)
+			if parts[0] == "tcp_port" {
```

Comment:
I don't like doing this, and would love to hear suggestions on a better way. The problem seems to be that custom attributes must be strings and not numbers, and since `tcp_port:6379` gets converted to a number the attribute isn't sent.